docs/tools/vdb_table/data/azureai.json

```diff
@@ -125,9 +125,9 @@
     "comment": ""
   },
   "disk_index": {
-    "support": "none",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Indexes are stored and managed on disk"
```

Comment:
Is there a URL to docs or any code that you can link in the `source_url` field for this? Disk-based indexes are something that could mean different things to different people.

Reply:
I guess this is not being claimed as it is set to none in the latest revision